books/free-programming-books-pt_BR.md

```diff
@@ -70,7 +70,9 @@
 
 #### Programação
 
+* [Algoritmos - Teoria e Prática](https://computerscience360.files.wordpress.com/2018/02/algoritmos-teoria-e-prc3a1tica-3ed-thomas-cormen.pdf) - Thomas H. Cormen, Charles E. Leiserson, Ronald L. Rivest, Clifford Stein (PDF)
```

Comment:
This appears not to be an authorized distribution. Please see CONTRIBUTING.md guidelines.